include/lsst/afw/coord/Observatory.h

```diff
@@ -47,19 +47,40 @@ namespace coord {
  */
 class Observatory {
 public:
-
+    
+    /**
+     * @brief Constructor for the observatory with lat/long as afwGeom::Angles
+     *
+     * @param[in] longitude  observatory longitude (positive values are E of Greenwich)
+     * @param[in] latitude  observatory latitude 
+     * @param[in] elevation  observatory elevation (meters above reference spheroid)
+     */
     Observatory(lsst::afw::geom::Angle const longitude, lsst::afw::geom::Angle const latitude, double const elevation);
-    Observatory(std::string const longitude, std::string const latitude, double const elevation);
 
+    /*
+     * @brief Constructor for the observatory with lat/long as strings
+     *
+     * @param[in] longitude  observatory longitude (dd:mm:ss.s, positive values are E of Greenwich)
+     * @param[in] latitude  observatory latitude  (dd:mm:ss.s)
+     * @param[in] elevation  observatory elevation (meters above reference spheroid)
+     *
+     */
+    Observatory(std::string const longitude, std::string const latitude, double const elevation);
```

Comment:
Old code. I'm not keen on constructing these from strings anyway. But adding & seems logical and I'll do it.